standalone/src/main/java/io/pravega/local/LocalPravegaEmulator.java

```diff
@@ -92,8 +94,9 @@ public void close() throws Exception {
 
     /**
      * Start controller and host.
+     * @throws Exception passes on the exception thrown by `inProcPravegaCluster`
      */
-    private void start() throws Exception {
+    public void start() throws Exception {
```

Comment:
1. Add `@VisibleForTesting` since the only reason this is non-private is because of unit tests.
2. Make package-private, not public (i.e., no visibility modifier). That should suffice.

Reply:
Fixed. Made it packave-private.